daml-lf/spec/transaction.rst

```diff
@@ -348,36 +344,28 @@ specification`_.
 
 ``contract_instance`` is required.
 
-Every element of ``stakeholders`` is a party identifier.
-``signatories`` must be a non-empty subset of ``stakeholders``.
+Every element of ``signatories`` and ``stakeholders`` is a party
+identifier.
 
 .. note:: *This section is non-normative.*
-
+	  
   The stakeholders of a contract are the signatories and the observers of
   said contract.
 
   The signatories of a contract are specified in the Daml-LF definition of
   the template for said contract. Conceptually, they are the parties that
   agreed for that contract to be created.
 
-``key_with_maintainers`` is optional. If present:
-
-* Its ``maintainers`` must be a subset of the ``signatories``;
```

Comment:
does this also count as validation or why did you remove it?

Reply:
This is part of validation, nothing constrains `maintainers` from serialization point of view.